xtask/src/task_clippy.rs

```diff
@@ -39,14 +39,17 @@ pub(crate) fn cmd_clippy(strict: bool, quiet: bool) -> Result<()> {
         run_clippy(&["-p", "propolis-server", "--features", "omicron-build"])?;
 
     // Check the Falcon bits
-    failed |= run_clippy(&[
-        "--features",
-        "falcon",
-        "-p",
-        "propolis-server",
-        "-p",
-        "propolis-client",
-    ])?;
+    #[cfg(target_os = "illumos")]
+    {
+        failed |= run_clippy(&[
+            "--features",
+            "falcon",
+            "-p",
+            "propolis-server",
+            "-p",
+            "propolis-client",
+        ])?;
+    }
```

Comment:
I believe this part of the change can be elided now that oxidecomputer/netadm-sys@d44d9e084f39e844f8083d4d9b39a331061ebbcc has landed?  (Provided a `cargo update -p libnet` is done)

Reply:
Removed